src/document/doc-builders.js

```diff
@@ -73,6 +73,7 @@ function indent(contents) {
  * @param {Doc} contents
  * @returns Doc
  */
+// eslint-disable-next-line prettier-internal-rules/no-identifier-n
 function align(n, contents) {
```

Comment:
Let's rename `n` to `widthOrString` here and in `doc-printer`. Just the variables, not the property.